src/System.IO.Pipes/src/System/IO/Pipes/PipeStream.Unix.cs

```diff
@@ -58,6 +59,15 @@ internal static string GetPipePath(string serverName, string pipeName)
             // naming scheme used as that breaks the ability for code running on an older
             // runtime to connect to code running on the newer runtime.  That means we're stuck
             // with a tmp file for the lifetime of the server stream.
+            if (isCurrentUserOnly)
+            {
+                string directory = Path.Combine(Path.GetTempPath(), $"CoreFx_{Interop.Sys.GetEUid()}");
```

Comment:
I don't know how strict we are, but we try to avoid interpolated strings ($"...") in product code, outside of asserts and rare events like maybe exception messages. The reason is that it does String.Format, which involves a STringBuilder, and walking through the string looking for markers. Just using `+` (or `Append`) does none of that work.

This probably isn't really a hot path but it's a good practice.

Reply:
same below

Reply:
I see. Didn't know that string interpolation was slower. Thanks for the advise. will fix.